File: relationship-manager/followup-orchestrator/src/main/java/com/bank/rm/followup/FollowUpOrchestratorApplication.java
package com.bank.rm.followup;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;
@SpringBootApplication(scanBasePackages = {"com.bank.rm.followup", "com.bank.rm.common"})
@EnableScheduling
public class FollowUpOrchestratorApplication {
    public static void main(String[] args) { SpringApplication.run(FollowUpOrchestratorApplication.class, args); }
}

File: relationship-manager/followup-orchestrator/src/main/kotlin/com/bank/rm/followup/FollowUpOrchestratorApplication.kt
package com.bank.rm.followup

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication(scanBasePackages = ["com.bank.rm.followup", "com.bank.rm.common"])
@EnableScheduling
class FollowUpOrchestratorApplication

fun main(args: Array<String>) {
    runApplication<FollowUpOrchestratorApplication>(*args)
}
